internal/handler/artwork.go

```diff
@@ -52,8 +52,15 @@ func GetArtworkByIDHandler(w http.ResponseWriter, r *http.Request) {
 func GetArtworksHandler(w http.ResponseWriter, r *http.Request) {
 	var err error
 	var page int //ページ番号
+	allowedSortColumns := map[string]bool{
+		"title":               true,
+		"artist_display_name": true,
+	}
 	pageStr := r.URL.Query().Get("page")
+	filterStr := r.URL.Query().Get("culture")
+	sortBy := r.URL.Query().Get("sort")
```

Comment:
あ、ごめんなさい一箇所だけ、クエリパラメータのキー名もsortByにしてほしいです！
修正していただいたらマージしていただいて大丈夫です！

```suggestion
	sortBy := r.URL.Query().Get("sortBy")
```